File: <100/072/072.py
import sympy.ntheory.factor_ as factor
import time

def main() -> None:
    upper_limit: int = 10**6+1
    totients: list   = (factor.totient(n) for n in range(2, upper_limit))
    answer: int      = sum(totients)
    print(answer)

    return


if __name__ == '__main__':
    start = time.time()
    main()
    elapsed = time.time() - start

    print(f'{elapsed} sec')

File: <100/072/072.jl
function main():: Nothing
  max_d:: Int  = 12000
  answer:: Int = 0
  for d ∈ max_d:-1:4
    n:: Int = d÷2 - 1 + (d%2)
    while 3*n > d
      answer += (gcd(n, d) == 1) ? 1 : 0
      n -= 1
    end
  end

  answer |> println
  return nothing
end


@time main()
